src/main/java/nl/tudelft/contextproject/model/level/util/Vec2I.java

```diff
@@ -0,0 +1,118 @@
+package nl.tudelft.contextproject.model.level.util;
+
+/**
+ * 2d integer vector class.
+ * Used for locations of objects of importance on the map.
+ * Mainly used in map generation.
+ */
+public class Vec2I {
+	
+	/**
+	 * The x coordinate of the vector.
+	 */
+	public int x;
```

Comment:
please change to yCoordinate or yCoord or something similair to avoid single letter fields